chapters/appendix-VIII-SPDX-Lite.md

```diff
@@ -0,0 +1,52 @@
+# Appendix-VIII SPDX Lite
+
+## 1. Explanation of SPDX Lite 
+
+The SPDX Lite defines the subset of the SPDX specification, from the point of view of use cases in industries. The SPDX Lite aims at the balance between the SPDX standard and actual workflows in industries. 
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Package information in the SDPX Lite is basic but indispensable for complying with licenses. It is easy to understand licensing information by reading an SPDX Lite file. It is easy to create manually an SPDX Lite file by anyone who does not have enough knowledge about licensing infomation, so that tools are not necessarily required to create an SPDX Lite file. 
+
+The SPDX Lite has the affinity with SPDX tools due to its containing the mandatory part of the Document Creation and Package Information in the SDPX Lite. 
+
+An SPDX Lite file can be used parallel with an SPDX file in software supply chains. 
+
+
+## 2. format of SPDX Lite
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Document Creation (those are Document Name, SPDX Document Namespace and Creator) is used for keeping the compatibility with SPDX tools.
+
+The main part of the Package Information (those are Package Name,  Package Version, Package File Name, Package Download Location,  Package Home Page, Concluded License, Declared License, Comments on License and Copyright Text) is used for exchanging license information.
+
+In the Package Information, Package SPDX Identifier and Files Analyzed are used for keeping the compatiblity with SPDX tools. 
+
+File Analyzed must be set to "false", when SPDX Lite is used.
+
+Other information (License Identifier, Extracted Text, License Name and License Comment) is used for exchanging license information.
+
+
+## Table of SPDX Lite
+
+| # | corresponding SPDX section no. | License Info. | Tag  | Rationale      |
+|:-----|:----|:--------|:-------|:------------------|
+|L1.1	 |2.4	 |Document Name	| DocumentName |To identify SPDX Lite document	|
+|L1.2	 |2.5	 |SPDX Document Namespace	| DocumentNamespace |To identify SPDX Lite document name	|
+|L1.3	 |2.8	 |Creator	      | Creator |To identify creator	|
+|L2.1	 |3.1	 |Package Name	| PackageName |To identify software	|
+|L2.2	 |3.2	 |Package SPDX Identifier	| SPDXID | To keep compatibility with SPDX. Feedback from SPDX team. |
+|L2.3	 |3.3	 |Package Version	| PackageVersion |To identify specific version	|
```

Comment:
Some of the fields listed here (such as PackageVersion and PackageFileName) are _not_ mandatory in SPDX itself; SPDX says they are optional.

Does SPDX-Lite make these fields mandatory? Or does SPDX-Lite say these fields are optional if they are optional in SPDX itself?

Reply:
The same question also applies to some of the other fields listed below that SPDX says are optional, not just to the two I mentioned.